### PR TITLE
[Added] get volumes data

### PR DESCRIPTION
**Pull request by dan12411**
新增 get volumes method & data model.

### CODE REVIEW

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/entities/Volume.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package net.makeagoodsoup.fugle_realtime_lib.core.entities
+
+data class Volume(
+    val price: Double, // 成交價
```
Comment (nick45chen) [inline fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/entities/Volume.kt:4]: @dan12411 不太確定是不是文件寫錯，price 是 String type

image

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt [hunk 2]
```diff
@@ -26,4 +23,10 @@ interface IntradayService {
         @Query("symbolId") symbolId: String,
         @Query("apiToken") apiToken: String
     ): Response<Intraday<DealtsData>>
+
+    @GET("intraday/volumes")
+    suspend fun getVolumes(
+        @Query("symbolId") symbolId: String,
+        @Query("apiToken") apiToken: String
```
Comment (nick45chen) [inline fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt:30]: @dan12411 缺少零股 parameter

image

**Review by nick45chen — CHANGES_REQUESTED** [review]

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt (hunk 2 above)
Comment (nick45chen) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt:30]: btw `oddLot` 為  optional parameter (有預設值)

Comment (nick45chen) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt:30]: nickpick:  補上文件註解說明 https://kotlinlang.org/docs/coding-conventions.html#file-annotations

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt [hunk 3]
```diff
@@ -44,4 +45,16 @@ class FugleHttpRepository {
             }
         }
     }
+
+    suspend fun getVolumes(symbolId: String, apiToken: String): Result<VolumesData> {
```
Comment (nick45chen) [inline fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt:49]: 同 https://github.com/Make-a-good-soup/fugle-realtime-kotlin/pull/6/files#r1091283638

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/entities/Volume.kt (hunk 1 above)
Comment (nick45chen) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/entities/Volume.kt:4]: 已確認是文件寫錯 😂

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt (hunk 2 above)
Comment (dan12411) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt]: 感謝 @nick45chen 已修正 [c29a850](https://github.com/Make-a-good-soup/fugle-realtime-kotlin/pull/6/commits/c29a850df88b61bdb2355ee91d892989b343ccae)

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt (hunk 3 above)
Comment (dan12411) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt]: 同上

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/entities/Volume.kt (hunk 1 above)
Comment (dan12411) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/entities/Volume.kt:4]: @nick45chen  讚讚 👍🏼

**Pull request merged**